**Background:**
Bombay hospitality Ltd. operates a franchise model for producing exotic Norwegian dinners throughout New England. The operating cost for a franchise in a week (W) is given by the equation W = $1,000 + $5X, where X represents the number of units produced in a week. Recent feedback from restaurant owners suggests that this cost model may no longer be accurate, as their observed weekly operating costs are higher.

**Objective:**

To investigate the restaurant owners' claim about the increase in weekly operating costs using hypothesis testing.


**Data Provided:**

•	The theoretical weekly operating cost model: W = $1,000 + $5X

•	Sample of 25 restaurants with a mean weekly cost of Rs. 3,050

•	Number of units produced in a week (X) follows a normal distribution with a mean (μ) of 600 units and a standard deviation (σ) of 25 units


In [22]:
import numpy as np
import scipy.stats as stats

### 1. State the Hypothesis
Null hypothesis (H0): The mean weekly operating cost equals the model value , i.e μ = 1000 + 5*600 = 4000.

Alternative hypothesis (Ha): The true mean weekly operating cost is greater than the model value , i.e μ > 4000.


In [23]:
# Given data
sample_mean=3050
n=25
std_dev=25
mean_units=600
alpha=0.05
#Hence:
pop_mean=4000   #1000+5*600=4000
pop_std=125     #5*25=125

### 2. Calculation of Test Statistics

Hence the population standard deviation (σ=125) is known and the sample size (n=25) is not much less than 30 , we will go for z-test.

In [24]:
z=(sample_mean-pop_mean) / (pop_std/np.sqrt(n))
print('The z score is' ,z)

The z score is -38.0


### 3. Determination of critical value

In [25]:
CI = 1-alpha
z_critical = stats.norm.ppf(CI)
print('z-critical value is',z_critical)

z-critical value is 1.6448536269514722


### 4. Decision Making


In [26]:
if z > z_critical:
    print ('Reject Null hypothesis (H0)')
else:
    print ('Fails to Reject Null hypothesis (H0)')


Fails to Reject Null hypothesis (H0)


### 5. Conclusion

- The hypothesis test was carried out to investigate whether the weekly operating costs are higher than what the theoretical model predicts.
- The calculated test statistic was -38.00, and the critical value at the 5% significance level was 1.645. 
- Hence, the test statistic is less than the critical value, we fail to reject the null hypothesis (H0).
- Since we failed to reject null hypothesi (H0) , there is not sufficient evidence present to support the restaurant owner's claim that the weekly operating costs are higher than the model suggests.